pygeoapi-config.yml

```diff
@@ -56,7 +56,7 @@ metadata:
             - data
             - api
         keywords_type: theme
-        terms_of_service: None
+        terms_of_service: null
```

Comment:
Except for the `null` and context def updates, can you revert the rest of the changes in this file?

Reply:
Yes, but can I request that we leave the hours of service one? I wanted to emphasise that the value there is not just unstructured text, but rather will be interpreted: https://schema.org/openingHours

Reply:
Also a value of `Lastname, Firstname` suggests a structure that isn't there, suggest changing that to `Full Name` as in this PR.

Reply:
Lastname, Firstname is as per ISO 19115 rules.  Is there a fullname property we can model after on schema.org?

Reply:
Ah, in that case `Lastname, Firstname` is fine. It's merely that https://schema.org/name is just unadulterated text, doesn't imply a person, etc. Happy to leave as is.